zetaclient/logs/fields.go

```diff
@@ -0,0 +1,33 @@
+package logs
+
+const (
+	// FieldModule is the field key for the module name in logs
+	FieldModule = "module"
```

Comment:
nit: no need to write comment for each field, it's possible to to comment the whole block above `const (`

Reply:
https://github.com/zeta-chain/node/pull/2628/commits/34cb11c4d14623805a52cbbc99c382f4cd781e77